### PR TITLE
Added template for app service web portal

### PR DESCRIPTION
**Pull request by Knyatsine**

### CODE REVIEW

File: azure/template.json [hunk 1]
```diff
@@ -46,12 +77,162 @@
...
+                                    "value": "[parameters('configNames')]"
+                                },
+                                {
+                                    "name": "Version",
+                                    "value": "[parameters('version')]"
+                                }
+                            ]
+                        }
+                    },
+                    "appServiceConnectionStrings": {
```
Comment (adam230594) [inline azure/template.json:203]: appServiceConnectionStrings as you already have appServiceConnectionStrings as a app setting anyway

File: azure/template.json [hunk 2]
```diff
@@ -11,6 +11,10 @@
         "serviceName": {
             "type": "string"
         },
+        "certificateName": {
```
Comment (adam230594) [inline azure/template.json:14]: Can this and its references be renamed to keyvaultCertificateName

File: azure/template.json [hunk 3]
```diff
@@ -11,6 +11,10 @@
...
+            "type": "string",
+            "defaultValue": ""
+        },
         "configNames": {
             "type": "string",
             "defaultValue": "__SolutionBaseName__"
```
Comment (adam230594) [inline azure/template.json:20]: This needs to be changed to the solution name

File: azure/template.json [hunk 4]
```diff
@@ -24,18 +28,45 @@
...
             "type": "string"
-        },        
+        },
         "configurationStorageConnectionString": {
             "type": "securestring"
         },
         "loggingRedisConnectionString": {
             "type": "securestring"
+        },
+        "sharedAppServicePlanResourceGroup": {
```
Comment (adam230594) [inline azure/template.json:38]: rename this to sharedEnvResourceGroup

File: azure/template.json [hunk 5]
```diff
@@ -46,12 +77,162 @@
...
+                    },
+                    "appServiceAppSettings": {
+                        "value": {
+                            "array": [
+                                {
+                                    "name": "ConfigurationStorageConnectionString",
+                                    "value": "[parameters('configurationStorageConnectionString')]"
+                                },
+                                {
+                                    "name": "ASPNETCORE_ENVIRONMENT",
```
Comment (adam230594) [inline azure/template.json:165]: Remove this. Most apps dont need this

File: azure/template.json [hunk 6]
```diff
@@ -46,12 +77,162 @@
...
+                                {
+                                    "name": "APPINSIGHTS_INSTRUMENTATIONKEY",
+                                    "value": "[reference('app-insights').outputs.InstrumentationKey.value]"
+                                },
+                                {
+                                    "name": "EnvironmentName",
+                                    "value": "[toUpper(parameters('environmentName'))]"
+                                },
+                                {
+                                    "name": "WEBSITE_SWAP_WARMUP_PING_PATH",
```
Comment (adam230594) [inline azure/template.json:185]: remove this as well

File: azure/template.json [hunk 7]
```diff
@@ -46,12 +77,162 @@
...
+                                    "value": "/ping"
+                                },
+                                {
+                                    "name": "WEBSITE_SWAP_WARMUP_PING_STATUSES",
```
Comment (adam230594) [inline azure/template.json:189]: remove this as well

File: azure/template.json (hunk 1 above)
Comment (Knyatsine) [reply azure/template.json]: @adam230594  corrected it

File: azure/template.json (hunk 2 above)
Comment (Knyatsine) [reply azure/template.json]: @adam230594  corrected it

File: azure/template.json (hunk 3 above)
Comment (Knyatsine) [reply azure/template.json]: @adam230594  corrected it

File: azure/template.json (hunk 4 above)
Comment (Knyatsine) [reply azure/template.json]: @adam230594  corrected it

File: azure/template.json (hunk 5 above)
Comment (Knyatsine) [reply azure/template.json]: @adam230594  corrected it

File: azure/template.json (hunk 6 above)
Comment (Knyatsine) [reply azure/template.json]: @adam230594  corrected it

File: azure/template.json (hunk 7 above)
Comment (Knyatsine) [reply azure/template.json]: @adam230594  corrected it

**Pull request merged**